clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/config/AwsProviderConfig.groovy

```diff
@@ -154,6 +155,10 @@ class AwsProviderConfig {
               amazonClientProvider, credentials, region.name, objectMapper, ctx
             )
           }
+
+          if (dynamicConfigService.isEnabled("aws.features.launch-templates", false)) {
```

Comment:
I'm not sure we have remote config wired in early enough that this will actually pick up a config change and add the agent (will still read the value from the `aws.featues.launch-templates.enabled` value in the spring environment, so probably worth having here in case we get earlier remote config, or for those that have some other remote config story)

Reply:
The FP property source is weaved in via an `EnvironmentPostProcessor` so its data is loaded plenty early.